src/modules/emote_autocomplete/twitch/EmoteAutocomplete.jsx

```diff
@@ -87,7 +87,7 @@ function patchEmoteImage(image, isConnected) {
     return;
   }
 
-  const url = String(image.srcset);
+  const url = String(image.hasAttribute('srcset') ? image.srcset : image.src);
```

Comment:
how do you feel about:

`image.srcset || image.src`

Reply:
dom nodes don't allow this

Reply:
```
> $0
< <img class="InjectLayout-sc-588ddc-0 iDjrEF tw-image tw-image-avatar" alt="User Avatar" src="https://static-cdn.jtvnw.net/jtv_user_pictures/night-profile_image-3588a4f04980d824-70x70.png">
> $0.srcset
< ''
> $0.src
< 'https://static-cdn.jtvnw.net/jtv_user_pictures/night-profile_image-3588a4f04980d824-70x70.png'
> $0.srcset || $0.src
< 'https://static-cdn.jtvnw.net/jtv_user_pictures/night-profile_image-3588a4f04980d824-70x70.png'
```